data/stoa0089/stoa002/__cts__.xml

```diff
@@ -3,8 +3,8 @@
             
   <ti:edition urn="urn:cts:latinLit:stoa0089.stoa002.perseus-lat2" workUrn="urn:cts:latinLit:stoa0089.stoa002" xml:lang="lat">
               <ti:label xml:lang="lat">de bello Gildonico</ti:label>
-    <ti:description xml:lang="eng">Claudian, with an English translation, in Two Volumes, Vol. I. Platnauer, Maurice, editor. Harvard University Press: Cambridge, MA, London, England. 1922 (printing).</ti:description>
-            </ti:edition>
+    <ti:description xml:lang="eng">Claudian. Claudian, with an English translation, Volume 1. Platnauer, Maurice, editor. Harvard University Press: Cambridge, MA, London, England. 1922 (printing).</ti:description>
```

Comment:
I don't think I would use "with an English translation" here. This is standard in all Loeb titles, and we don't consider it part of the main title (at least I don't recall using it that way — others may have.)

Ideally, this would read, 

`<ti:description xml:lang="eng">Claudian, Volume 1. Platnauer, Maurice, editor. Harvard University Press: Cambridge, MA; London, England, 1922.</ti:description>`

Reply:
OK, I was thinking that as well, they had also had in Two Volumes in the title as well. I will in general not include "with an English translation" in the title.